### PR TITLE
Added Sorting Algorithm Visulizer Using Pygame

### PR DESCRIPTION
**Pull request by Ctoic**
Also implemented Likedlist in python.

### CODE REVIEW

File: Sorting_visualization/Sort_vi.py [hunk 1]
```diff
@@ -0,0 +1,595 @@
+import pygame
```
Comment (ShehrozIrfan) [inline Sorting_visualization/Sort_vi.py]: Please add this file in the right folder:
`Python/Sorting_visualization/Sort_vi.py`

**Review by ShehrozIrfan — CHANGES_REQUESTED** [review]
Overall, looks good.
Just need to make a minor change for folder structure.

**Comment by Ctoic** [comment]
I have now moved the code to the Python directory please check and merge it.

**Comment by Ctoic** [comment]
Dear @ShehrozIrfan  kindly merge my pull request. I have added the directory in python.

**Review by ShehrozIrfan — APPROVED** [review]
LGTM

**Comment by ShehrozIrfan** [comment]
Great 🚀

Have fun!

**Pull request merged**